examples/aws_efa/README.md

```diff
@@ -0,0 +1,98 @@
+# Using AWS Elastic Fabric Adapter (EFA) with SkyPilot
+
+## What is AWS EFA?
+
+Elastic Fabric Adapter (EFA) is a network interface for Amazon EC2 instances that enables high-performance computing and machine learning applications to scale efficiently in the cloud. EFA provides lower latency, higher throughput, and OS-bypass capabilities that are critical for communication-intensive workloads such as distributed training.
+
+
+### Setting up EFA cluster 
+For complete details on setting up EFA with Amazon EKS, refer to the [official AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/node-efa.html).
+
+
+To check if EFA is enabled, run 
+```
+kubectl get nodes "-o=custom-columns=NAME:.metadata.name,INSTANCETYPE:.metadata.labels.node\.kubernetes\.io/instance-type,GPU:.status.allocatable.nvidia\.com/gpu,EFA:.status.allocatable.vpc\.amazonaws\.com/efa"
+```
+you can expect a sample output like
+```
+NAME                           INSTANCETYPE    GPU   EFA
+hyperpod-i-055aeff9546187dee   ml.g5.8xlarge   1     1
+hyperpod-i-09662f64f615c96f5   ml.g5.8xlarge   1     1
+hyperpod-i-099e2a84aba621d52   ml.g5.8xlarge   1     1
+...
+```
+
+### Kubernetes-Specific Configuration
+This example demonstrates how to configure SkyPilot to use EFA for NCCL performance testing on AWS. Let's break down the `nccl_efa.yaml` file component by component:
+
+```yaml
+config:
+  kubernetes:
+    pod_config:
+      spec:
+        containers:
+        - resources:
+            limits:
+              vpc.amazonaws.com/efa: 2
+            requests:
+              vpc.amazonaws.com/efa: 2
+```
+
+This section is important for EFA integration:
+
+- `config.kubernetes.pod_config`: Provides Kubernetes-specific pod configuration
+- `spec.containers[0].resources`: Defines resource requirements
+  - `limits.vpc.amazonaws.com/efa: 2`: Limits the pod to using 2 EFA devices
+  - `requests.vpc.amazonaws.com/efa: 2`: Requests 2 EFA devices for the pod
+
+The `vpc.amazonaws.com/efa` resource type is exposed by the AWS EFA device plugin in Kubernetes. 
+
+### Basic Task Configuration
+
+```yaml
+name: nccl-test
+
+resources:
+  cloud: kubernetes
+  accelerators: A100:2
+  cpus: 64
+  image_id: docker:public.ecr.aws/hpc-cloud/nccl-tests:latest
+
+num_nodes: 2
+```
```

Comment:
How about include the setup/run section here as well, to make it more complete

Reply:
Updated